README.md

```diff
@@ -79,73 +97,106 @@ Fields that can be included in the response body for `GET /`.
+STAPI follow the modern web API practices of using HTTP Request Methods
+("verbs") and the `Content-Type` header to drive behavior on resources
+("nouns") in the endpoints listed below.
+
+The following table describes the service resources available in a STAPI
+implementation that supports all three of the foundation specifications. Note
+that the 'Endpoint' column is more of an example in some cases.
+
+| Endpoint | Specified in | Accepts | Returns | Description |
+| -------- | ------------ | ------- | ------- | ----------- |
+| `GET /` | Core | - | [Landing Page](#landing-page) | |
+| `GET /conformance` | Core | - | Conformance Classes | |
+| `GET /products` | Core | - | [Products Collection](./product/README.md) | Figure out which constraints are available for which `productId` |
+| `GET /products/{productId}` | Core | - | [Product](./product/README.md) | |
+| `GET /products/{productId}/constraints` | Core | - | JSON Schema | |
+| `GET /products/{productId}/order-parameters` | Core | - | JSON Schema | |
+| `GET /orders` | Core | - | [Orders Collection](./order/README.md#order-collection) | |
+| `GET /orders/{orderId}` | Core | - | [Order Object](./order/README.md#order-object) | |
+| `POST /products/{productId}/orders` | Core | [Order Request](./order/README.md#order-request) | - | Order a capture with a particular set of [parameters](https://github.com/Element84/stapi-spec/blob/main/product/README.md#parameters) as defined in the products or a request that was provided through the opportunities endpoint. |
+| `POST /products/{productId}/opportunities` | Opportunities | [Opportunity Request](./opportunity/README.md#opportunity-request) | Sync search: [Opportunities Collection](./opportunity/README.md#opportunities-collection); Async search: [Opportunity Search Record](./opportunity/README.md#opportunity-search-record) | Explore the opportunities available for a particular set of [parameters](https://github.com/Element84/stapi-spec/blob/main/product/README.md#parameters) |
+| `GET /products/{productId}/opportunities/{opportunityCollectionId}` | Opportunities (Async) | - | [Opportunities Collection](./opportunity/README.md#opportunities-collection) | Get the opportunity collection for an async opportunity search |
+| `GET /searches/opportunities` | Opportunities (Async) | - | [[Opportunity Search Record](./opportunity/README.md#opportunity-search-record)] | |
+| `GET /searches/opportunities/{searchRecordId}` | Opportunities (Async) | - | [Opportunity Search Record](./opportunity/README.md#opportunity-search-record) | |
+| `GET /searches/opportunities/{searchRecordId}/statuses` | Opportunities (Async) | - | [[Opportunity Search Status](./opportunity/README.md#opportunity-search-status)] | |
+| `POST /searches/opportunities/{searchRecordId}/statuses` | Opportunities (Async) | [Opportunity Search Status](./opportunity/README.md#opportunity-search-status) | - | POST a search status update. Returns an empty 201 on success. |
```

Comment:
The whole statuses piece here borrows heavily from the `order-statuses` extension. However, the spec does not outline an equivalent of this endpoint for order statuses. I am uncertain if this is because it is intended to be an implementation-specific endpoint and not addressable by the spec or if it is an oversight.

If I should remove this endpoint from the spec let me know. Similarly, if we need to add an endpoint to update order statuses let me know that too and I can see about getting a change in for that (or at least an issue).